<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>379</width>
    <height>390</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="sBusTab">
       <attribute name="title">
        <string>sBus Decoder</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QFrame" name="frame_2">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Output:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Channel</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Frame Rate</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Failsafe</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QFrame" name="frame">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>1:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="out1Channel">
             <item>
              <property name="text">
               <string>1</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>2</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>3</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>4</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>5</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>6</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>7</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>8</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>9</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>10</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>11</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>12</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>13</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>14</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>15</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>16</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>17</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>18</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="out1FrameRate">
             <item>
              <property name="text">
               <string>50 Hz</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>100 Hz</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>150 Hz</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="out1Failsafe">
             <item>
              <property name="text">
               <string>Last</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Off</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Neutral</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QFrame" name="frame_3">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>2</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="out2Channel">
             <item>
              <property name="text">
               <string>1</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>2</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>3</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>4</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>5</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>6</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>7</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>8</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>9</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>10</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>11</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>12</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>13</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>14</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>15</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>16</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>17</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>18</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="out2FrameRate">
             <item>
              <property name="text">
               <string>50 Hz</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>100 Hz</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>150 Hz</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="out2Failsafe">
             <item>
              <property name="text">
               <string>Last</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Off</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Neutral</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QFrame" name="frame_4">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>3:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="out3Channel">
             <item>
              <property name="text">
               <string>1</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>2</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>3</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>4</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>5</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>6</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>7</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>8</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>9</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>10</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>11</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>12</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>13</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>14</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>15</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>16</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>17</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>18</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="out3FrameRate">
             <item>
              <property name="text">
               <string>50 Hz</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>100 Hz</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>150 Hz</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="out3Failsafe">
             <item>
              <property name="text">
               <string>Last</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Off</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Neutral</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QFrame" name="frame_5">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>4:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="out4Channel">
             <item>
              <property name="text">
               <string>1</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>2</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>3</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>4</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>5</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>6</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>7</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>8</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>9</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>10</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>11</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>12</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>13</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>14</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>15</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>16</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>17</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>18</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="out4FrameRate">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <item>
              <property name="text">
               <string>50 Hz</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>100 Hz</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>150 Hz</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="out4Failsafe">
             <item>
              <property name="text">
               <string>Last</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Off</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Neutral</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QFrame" name="frame_6">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="readPushButton">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Read</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="writePushButton">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Write</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="servoSequencerTab">
       <attribute name="title">
        <string>Servo Sequencer</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>379</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuPort">
    <property name="title">
     <string>Port</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuPort"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
